### PR TITLE
Css rewrite

### PR DESCRIPTION
**Pull request by Synicix**
- Partial rewrite of some CSS to fix odd behavior with scaling and displaying elements. Also fix serval scrolling issues with elements
- Fixed flexbox grow issues with safari by adding prefix.

Before tables like this couldn't scroll on safari
image

With the new changes, the view on safari now matches the original design intended.
image

### CODE REVIEW

**Comment by Synicix** [comment]
Requesting review on this due to Safari complaints

**Comment by Synicix** [comment]
@mahos Can I get a review on this, would like to release these fixes at some point.

**Comment by mahos** [comment]
> @mahos Can I get a review on this, would like to release these fixes at some point.

Hey @Synicix sorry about the delay - just checked it out and I'm still having trouble scrolling down on safari for larger tables - not sure if you intended to fix that issue here though, so let me know if I should just approve it, and we work on fixing that safari scroll issue for large tables later on.

**Review by mahos — APPROVED** [review]

File: src/Components/MainTableView/TableContent.css [hunk 1]
```diff
@@ -1,12 +1,13 @@
...
   height: 100%;
+  width: 100%;
 }
-
 .table-content-viewer {
-  position: relative;
   display: flex;
   flex-direction: column;
-  box-sizing: border-box;
+  box-sizing: border-box;
```
Comment (guzman-raphael) [inline src/Components/MainTableView/TableContent.css:10]:
```suggestion
  box-sizing: border-box;
```

**Review by guzman-raphael — CHANGES_REQUESTED** [review]
@Synicix Can you also add some POW stills/GIF's to the PR description for this fix?

**Comment by Synicix** [comment]
@guzman-raphael Added pictures

**Comment by guzman-raphael** [comment]
@Synicix thanks. Is this one now ready to go? I think mentioning patching another bug but it would be in another PR?

**Comment by Synicix** [comment]
Yeah, anything that is extra will be on a different PR

**Pull request merged**